apache-hbase-mixin/signals/regionserver.libsonnet

```diff
@@ -0,0 +1,398 @@
+        type: 'raw',
+        description: 'The rate of slow put operations.',
+        unit: 'ops',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (rate(server_slow_put_count{%(queriesSelector)s}[$__rate_interval]))',
+            legendCustomTemplate: 'put',
+          },
+        },
+      },
+
+      slowDeleteRate: {
+        name: 'Slow deletes',
+        type: 'raw',
+        description: 'The slow delete operations.',
+        unit: 'ops',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (rate(server_slow_delete_count{%(queriesSelector)s}[$__rate_interval]))',
+            legendCustomTemplate: 'delete',
+          },
+        },
+      },
+
+      slowGetRate: {
+        name: 'Slow gets',
+        type: 'raw',
+        description: 'The rate of slow get operations.',
+        unit: 'ops',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (rate(server_slow_get_count{%(queriesSelector)s}[$__rate_interval]))',
+            legendCustomTemplate: 'get',
+          },
+        },
+      },
+
+      slowIncrementRate: {
+        name: 'Slow increments',
+        type: 'raw',
+        description: 'The rate of slow increment operations.',
+        unit: 'ops',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (rate(server_slow_increment_count{%(queriesSelector)s}[$__rate_interval]))',
+            legendCustomTemplate: 'increment',
+          },
+        },
+      },
+
+      // Cache metrics
+      blockCacheHitPercent: {
+        name: 'Block cache hit percentage',
+        type: 'gauge',
+        description: 'The percent of time that requests hit the cache.',
+        unit: 'percent',
+        sources: {
+          prometheus: {
+            expr: 'server_block_cache_express_hit_percent{%(queriesSelector)s}',
+            legendCustomTemplate: '{{instance}}',
+          },
+        },
+      },
+
+      // JVM memory for RegionServers
+      jvmHeapMemoryUsage: {
+        name: 'RegionServer JVM heap memory usage',
+        type: 'raw',
+        description: 'Heap memory usage for the RegionServer JVM.',
+        unit: 'percentunit',
+        sources: {
+          prometheus: {
+            expr: 'jvm_metrics_mem_heap_used_m{%(queriesSelector)s, processname="RegionServer"} / clamp_min(jvm_metrics_mem_heap_committed_m{%(queriesSelector)s, processname="RegionServer"}, 1)',
+            legendCustomTemplate: '{{instance}}',
+          },
+        },
+      },
+
+      regionServerAuthenticationSuccess: {
+        name: 'RegionServer authentication successes',
+        type: 'raw',
+        description: 'The rate of successful authentications to the RegionServer.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (rate(region_server_authentication_successes{%(queriesSelector)s}[$__rate_interval]))',
```

Comment:
This is a raw metric that uses a rate function correctly but it's also a duplicate of the same signal and same metric in `cluster.libsonnet` file but that one doesn't use the rate function as this one. Can we remove it from the clusters signals?